Directory.Build.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
     <LangVersion>10.0</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
When I've tried to use driver's package from myget (the version did not exist on nuget) without adjusting nuget sources the current version simply resolves any closest version of driver available instead of failing.

Reply:
Can you add an explanation on how is this change related to [CSHARP-5117](https://jira.mongodb.org/browse/CSHARP-5117) please?